pkg/scheduler/scheduler.go

```diff
@@ -291,6 +291,8 @@ func New(client clientset.Interface,
 		nodeInfoSnapshot:               snapshot,
 	}
 
+	metrics.Register()
```

Comment:
why moving this to here makes a difference?

Reply:
oh, because it has to be done before initializing the scheduling_queue

Reply:
Yes, it is the reason.  I wanted to find a more essential & safe way to `Register` metric instances, though.